virttest/qemu_storage.py

```diff
@@ -345,10 +346,10 @@ def snapshot_del(self, blkdebug_cfg=""):
                                        " parameters")
         if blkdebug_cfg:
             cmd += " blkdebug:%s:%s" % (blkdebug_cfg, self.image_filename)
+            return process.run(cmd, ignore_status=True)
```

Comment:
OK, I was completely confused this PR and autotest/tp-qemu#654.
IMO,
image_io.snapshot_del(blkdebug_cfg=blkdebug_cfg) won't trigger any CmdErrror **with this PR**.
It's meaningless that catch Exception **in testcase** for it.

Maybe some guys in redhat understand all of it and can review this.
Let's wait for their's response.